src/main/kotlin/no/nav/syfo/behandlendeenhet/api/TildelHistorikkResponseDTO.kt

```diff
@@ -0,0 +1,66 @@
+package no.nav.syfo.behandlendeenhet.api
+
+import no.nav.syfo.behandlendeenhet.EnhetService.Companion.SYSTEM_USER_IDENT
+import no.nav.syfo.behandlendeenhet.api.TildelOppfolgingsenhetHistorikkType.*
+import no.nav.syfo.behandlendeenhet.domain.Oppfolgingsenhet
+import java.time.LocalDateTime
+
+data class TildelHistorikkResponseDTO(
+    val oppfolgingsenheter: List<TildelOppfolgingsenhetHistorikkDTO>,
+)
+
+enum class TildelOppfolgingsenhetHistorikkType {
+    TILDELT_ANNEN_ENHET_AV_VEILEDER,
+    TILDELT_TILBAKE_TIL_ENHET_AV_VEILEDER,

```

Comment:
Burde vi være tydeligere her? 😊
```suggestion
    TILDELT_TILBAKE_TIL_GEOGRAFISK_ENHET_AV_VEILEDER,

```

Reply:
Ja, det vil jo gjøre det enda tydeligere 😊